opentreemap/importer/templates/importer/partials/species_resolve_popover.html

```diff
@@ -1,11 +1,13 @@
 {% extends "treemap/partials/popover.html" %}
+{% load i18n %}
 {% load l10n %}
 
-{% block title %}Species Field Error{% endblock title %}
+{% block title %}{% trans "Species Field Error" %}{% endblock title %}
 
 {% block content %}
+{# these are already translated on the server side: #}
 <p>{{ field.msg }}</p>
-<p>Did you mean: "{{ field.custom_resolver.suggestion.display_name }}"?</p>
+<p>{% trans "Did you mean:" %} "{{ field.custom_resolver.suggestion.display_name }}"?</p>
```

Comment:
This still is a good candidate for `blocktrans`, since the `?` should translated as well. (So we can include `¿` for spanish).  Can you change this to something like:
```html
{% with display_name=field.custom_resolver.suggestion.display_name %}
<p>{% blocktrans %}Did you mean: "{{ display_name }}"?{% endblocktrans %}</p>
{% endwith %}
```